python/cuda_parallel/cuda/parallel/experimental/_bindings.pyx

```diff
@@ -1609,3 +1609,185 @@ cdef class DeviceUniqueByKeyBuildResult:
                 f"Failed executing unique_by_key, error code: {status}"
             )
         return storage_sz
+
+
+# --------------------------------------------
+#   DeviceUnaryTransform/DeviceBinaryTransform
+# --------------------------------------------
+cdef extern from "cccl/c/transform.h":
+    cdef struct cccl_device_transform_build_result_t:
+        pass
+
+    cdef CUresult cccl_device_unary_transform_build(
+        cccl_device_transform_build_result_t *build_ptr,
+        cccl_iterator_t d_in,
+        cccl_iterator_t d_out,
+        cccl_op_t op,
+        int, int, const char *, const char *, const char *, const char *
+    ) nogil
+
+    cdef CUresult cccl_device_unary_transform(
+      cccl_device_transform_build_result_t build,
+      cccl_iterator_t d_in,
+      cccl_iterator_t d_out,
+      unsigned long long num_items,
```

Comment:
In other algorithms `uint64_t` is used as `num_items` type. Can we do the same in transform as well?